youtube_dl/extractor/dreisat.py

```diff
@@ -1,193 +1,29 @@
+# coding: utf-8
 from __future__ import unicode_literals
 
-import re
+from .zdf import ZDFIE
+
+
+class DreiSatIE(ZDFIE):
+    _VALID_URL = r'https?://www\.3sat\.de/(?:[^/]+/)*(?P<id>[^/?]+)\.html'
+    _TESTS = [{
+         'url': 'https://www.3sat.de/dokumentation/natur/dolomiten-sagenhaftes-juwel-der-alpen-100.html',
+         'info_dict': {
```

Comment:
If this passes the test, then: Nice! I have a different test case, but it's probably not worth the redundancy.